### PR TITLE
Create voting-process.md (Pending votes)

### PR DESCRIPTION
**Pull request by staltz**

### CODE REVIEW

**Comment by Powersource** [comment]
Probably going to have to be updated in the future, but so far a `y` from me

**Comment by Powersource** [comment]
I think this can close https://github.com/scuttlebutt-eu/important-documents/issues/20 as well

**Comment by elavoie** [comment]
I think a change of criteria or the meaning of existing criteria of membership is sufficiently fundamental to require an explicit 'Yes' from at least 50% + 1 of the current members. For other changes that don't change the meaning of the rules (ex: clarifications, typos), I don't think the 50% rule is necessary.

Other than that I agree that we can use the YW?! process for membership changes.

**Comment by cblgh** [comment]
@elavoie sorry, i'm not sure i understand what you intend with the highlighted portion of the below paragraph

> **I think a change of criteria or the meaning of existing criteria of membership is sufficiently fundamental** to require an explicit 'Yes' from at least 50% + 1 of the current members. For other changes that don't change the meaning of the rules (ex: clarifications, typos), I don't think the 50% rule is necessary.

can you rephrase that to make that clearer? or let me know briefly on mumble during today's what you intend :3

**Comment by elavoie** [comment]
@cblgh The comment replies to https://github.com/scuttlebutt-eu/important-documents/pull/26#issuecomment-612854909, but it probably would have been clearer if it had been put in issue #20. 

As for the meaning, the voting process only requires a single Yes vote to pass, all others can be Whatever. For Membership changes, I think this is too lax and we should use a strict majority of Yes.

**Comment by ZELFs** [comment]
Vote results:
@staltz: y
@arj03: y
@elavoie : y 
@Powersource : y 
me: y
@keks: pending 
@cblgh: pending (optional)

**Comment by keks** [comment]
y
